include/RoboCupGameControlData.h

```diff
@@ -78,8 +78,10 @@ struct RobotInfo
 {
   uint8_t penalty;              // penalty state of the player
   uint8_t secsTillUnpenalised;  // estimate of time till unpenalised
-  uint8_t yellowCardCount;  // number of yellow cards
-  uint8_t redCardCount;  // number of red cards
+  uint8_t numberOfWarnings;     // number of warnings
```

Comment:
Mhm, maybe the variable name`warningCount` would fit better with the naming scheme of the other variables.